server/src/main.rs

```diff
@@ -87,11 +87,7 @@ impl LanguageServer for Backend {
     }
 
     async fn initialized(&self, _: InitializedParams) {
-        // Update the configuration sets by fetching the configurations from the client. This is performed after
-        // initialization because the client may not be ready to provide configurations before initialization.
-        self.session.fetch_configurations(&self.client).await;
-
-        // Publish the diagnostics for all files
+        // We wait until after the server and client are fully initialized to publish any diagnostics we've found.
```

Comment:
Should this comment be on `initialize`?

Reply:
I guess it should be on both.
But you're right, with the tense it's in, it should be on `initialize`.